client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.tsx

```diff
@@ -332,6 +309,39 @@ function CheckoutSummaryPlanFeatures() {
 	);
 }
 
+function CheckoutSummaryAnnualUpsell( props: {
+	plan: ResponseCartProduct;
+	onChangePlanLength: ( uuid: string, productSlug: string, productId: number ) => void;
+} ) {
+	const translate = useTranslate();
+	const productSlug = props.plan?.product_slug;
+
+	if ( ! productSlug || ! isWpComPlan( productSlug ) ) {
+		return null;
+	}
+
+	return (
+		<CheckoutSummaryFeatures>
+			<CheckoutSummaryFeaturesTitle>
+				{ translate( 'Included with an annual plan' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( 'Included with annual plans' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/checkout-domain-credit-feature)